src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
```

Comment:
Use desturctuirng to avoid writing `person.` everytime

Reply:
I prefer to destruct inside component